paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutInteractor.kt

```diff
@@ -309,10 +309,6 @@ internal class DefaultPaymentMethodVerticalLayoutInteractor(
             return PaymentMethodVerticalLayoutInteractor.SavedPaymentMethodAction.NONE
         }
 
-        if (!canEdit && !canRemove) {
-            return PaymentMethodVerticalLayoutInteractor.SavedPaymentMethodAction.NONE
-        }
-
```

Comment:
Same here, it's possible to have multiple non-modifiable payment methods but be able to remove none of them.